src/main/client/webview/index.ts

```diff
@@ -325,7 +325,11 @@ export function useWebview(path = 'http://assets/webview/index.html') {
     }
 
     function onSyncedMetaChange(object: alt.Object, key: string, newValue: any) {
-        if (native.isEntityAPed(object.scriptID) && object.scriptID === alt.Player.local.scriptID) {
+        if (
+            typeof object.scriptID === 'number' &&
```

Comment:
![Screenshot 2024-07-25 181123](https://github.com/user-attachments/assets/6dbc7428-8390-427e-98a2-86592f35706f)
Got this issue somehow, fixed with this line